openmetadata-service/src/main/java/org/openmetadata/service/search/SearchRepository.java

```diff
@@ -312,7 +312,7 @@ public void updateEntity(EntityInterface entity) {
           scriptTxt = getScriptWithParams(entity, doc);
         } else {
           SearchIndex elasticSearchIndex = searchIndexFactory.buildIndex(entityType, entity);
-          doc = elasticSearchIndex.buildESDoc();
+          doc = elasticSearchIndex.buildElasticSearchDocument();
```

Comment:
why are we suing elasticSearchIndex naming here, lets go to generic name searchIndex

Reply:
done @harshach